modules/home-manager/fzf.nix

```diff
@@ -0,0 +1,34 @@
+{ config
+, lib
+, sources
+, ...
+}:
+let
+  cfg = config.programs.fzf.catppuccin;
+  enable = cfg.enable && config.programs.fzf.enable;
+  palette = (lib.importJSON "${sources.palette}/palette.json").${cfg.flavour}.colors;
+in
+{
+  options.programs.fzf.catppuccin =
+    lib.ctp.mkCatppuccinOpt "fzf";
+
+  config.programs.fzf.colors = lib.mkIf enable
+    # Manually populate with colors from catppuccin/fzf
+    (lib.attrsets.mapAttrs (_: color: palette.${color}.hex)
+      {
+        "bg+" = "surface0";
+        bg = "base";
+        spinner = "rosewater";
+        hl = "red";
```

Comment:
nit: group hl with hl+ and the same for fg and fg+ like you did for bg and bg+

Reply:
I can do that, but first I should mention I put the options in that specific order to match the (seemingly arbitrary) order of the options in catppuccin/fzf to make it easier to compare.

Reply:
Ahhh, that makes total sense.